Heavy Rewording of Section For Clarity (#2362)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/create-a-phone-system-call-queue.md

![Icon of the number 1, referencing a callout in the previous screenshot](media/sfbcallout1.png)

> You can select up to 200 call agents who either belong to specified mailing lists or groups. Call agents selected must  **either** be online users with a **Phone System** license and Enterprise Voice enabled **or** have a Calling Plan.

Author: ```suggestion
You can select up to 200 call agents who belong to any of the following mailing lists or groups:

- Office 365 group
- Security group
- Distribution list

Call agents selected must  **either** be online users with a **Phone System** license and Enterprise Voice enabled **or** have a Calling Plan.
```

> You can select up to 200 call agents who either belong to specified mailing lists or groups. Call agents selected must either be online users with a **Phone System** license as well as have Enterprise Voice enabled **or** have a Calling Plan.

Reviewer: ```suggestion
You can select up to 200 call agents who either belong to specified mailing lists or groups. Call agents selected must  **either** be online users with a **Phone System** license and Enterprise Voice enabled **or** have a Calling Plan.
```

  > [!NOTE]

>   > This also applies if you want to redirect calls to people in your organization who are online. These individuals must have a **Phone System** license and Enterprise Voice enabled **or** have a Calling Plan. See [Assign Skype for Business licenses](/skypeforbusiness/skype-for-business-and-microsoft-teams-add-on-licensing/assign-skype-for-business-and-microsoft-teams-licenses), [Assign Microsoft Teams licenses](assign-teams-licenses.md), or [Which Calling Plan is right for you?](https://docs.microsoft.com/microsoftteams/calling-plan-landing-page) for more information.

Reviewer 2: ```suggestion
  > This also applies if you want to redirect calls to people in your organization who are online. These individuals must have a **Phone System** license and Enterprise Voice enabled **or** have a Calling Plan. See [Assign Skype for Business licenses](https://docs.microsoft.com/en-us/skypeforbusiness/skype-for-business-and-microsoft-teams-add-on-licensing/assign-skype-for-business-and-microsoft-teams-licenses), [Assign Microsoft Teams licenses](https://docs.microsoft.com/en-us/microsoftteams/assign-teams-licenses), or [Which Calling Plan is right for you?](https://docs.microsoft.com/microsoftteams/calling-plan-landing-page) for more information.
```

>   > This also applies if you want to redirect calls to people in your organization who are online. These individuals must have a **Phone System** license and Enterprise Voice enabled **or** have a Calling Plan. See [Assign Skype for Business licenses](https://docs.microsoft.com/en-us/skypeforbusiness/skype-for-business-and-microsoft-teams-add-on-licensing/assign-skype-for-business-and-microsoft-teams-licenses), [Assign Microsoft Teams licenses](https://docs.microsoft.com/en-us/microsoftteams/assign-teams-licenses), or [Which Calling Plan is right for you?](https://docs.microsoft.com/microsoftteams/calling-plan-landing-page) for more information.

Reviewer: ```suggestion
  > This also applies if you want to redirect calls to people in your organization who are online. These individuals must have a **Phone System** license and Enterprise Voice enabled **or** have a Calling Plan. For more information see [Assign Skype for Business licenses](https://docs.microsoft.com/skypeforbusiness/skype-for-business-and-microsoft-teams-add-on-licensing/assign-skype-for-business-and-microsoft-teams-licenses), [Assign Microsoft Teams licenses](https://docs.microsoft.com/microsoftteams/assign-teams-licenses), or [Which Calling Plan is right for you?](https://docs.microsoft.com/microsoftteams/calling-plan-landing-page)
```

 To enable an agent for Enterprise Voice, you can use Windows PowerShell. For example run:  `Set-CsUser -identity "Amos Marble" -EnterpriseVoiceEnabled $true`

> - Online users with a **Phone System** license or a Calling Plan that are added to either: an Office 365 Group, a mail-enabled Distribution List, or a Security Group. It might take up to 3 hours for a newly added agent in a distribution list or a security group to start receiving calls from a call queue. A newly created distribution list or security group might take up to 48 hours to become available to be used with call queues. Newly created Office 365 Groups are available almost immediately.

Reviewer: ```suggestion
- Online users with a **Phone System** license or a Calling Plan that are added to either: an Office 365 Group, a mail-enabled Distribution List, or a Security Group. It might take up to three hours for a newly added agent in a distribution list, or a security group, to start receiving calls from a call queue. A newly created distribution list or security group might take up to 48 hours to become available to be used with call queues. Newly created Office 365 Groups are available almost immediately.
```
